docs/setup/settings.asciidoc

```diff
@@ -26,11 +26,11 @@ Set to `false` to disable Console. *Default: `true`*
 Add sources for the https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src[Content Security Policy `script-src` directive].
 
 `csp.disableUnsafeEval`::
-experimental[] Set this to `true` to remove the https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#unsafe_eval_expressions[`unsafe-eval`] source expression from the `script-src` directive. *Default: `false`*
+Set this to `false` to add the https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#unsafe_eval_expressions[`unsafe-eval`] source expression to the `script-src` directive. *Default: `true`*
```

Comment:
Removing the `experimental` flag means we're committing to support the option for the foreseeable future. Remember that our deprecation strategy has changed to long-term support (~ 18 months) so we need to be absolutely certain we want to retain the option.
If, OTOH, you feel strongly about changing the option name from `disableUnsafeEval` to `enableUnsafeEval`, then you could change `experimental` to `deprecated`. Ideally, we should add a date-to-be-removed (or a version) when the option is going to go away.

Reply:
While I am a fan of boolean values being named to reflect true = on/enabled, I think I agree with Tina that if it is already in use (widely used?) it probably doesn't make sense to rename it.

If usage is minimum and we'll be supporting the option long term I am more inclined to say proceed with the rename. Is the option no longer really needed/likely to be deprecated in the future, now that we can support inline partials? Or do we see this as an option that will always be around regardless? That may help decide if the rename is worth it.

Reply:
It's not _needed_ to change the option name, but I just find it confusing that it's called `disableUnsafeEval` when the default value is now going to be `true`. So if a user is ever going to touch this setting it's to set it to `false`, which is going to look like `csp.disableUnsafeEval: false`... parsing that just takes a little bit more brain power - at least for me. But maybe it's lack for sleep 😅

@elastic/kibana-security: What do you think?

Reply:
Good point about `csp.script_src`. Maintaining `csp.disableUnsafeEval` was my intention, just deprecating it. Would you be ok with that?

Reply:
> Maintaining csp.disableUnsafeEval was my intention, just deprecating it. Would you be ok with that?

Yup!

Reply:
Ok, here's my proposal trying to take everybody's feedback into account:

1. We keep `csp.disableUnsafeEval` (no rename), but change it to default to `true` instead of `false`.
2. We deprecate it and ask people to use `csp.script_src: ['unsafe-eval']` instead if they wish to use `unsafe-eval`.

Regarding depression date/version, we can pick some reasonable time/version in the future. Since it doesn't cost us much to keep it I don't mind making it long. But since this is currently experimental and that we're literally changing the default behaviour to be something that most users would have used this config option to specifically set, we could probably also make it a lot shorter - e.g. next minor version - but other suggestions are welcome as well.

Suggestion:

```suggestion
deprecated:[8.7.0,Use `csp.script_src: ['unsafe-eval']` instead if you wish to enable `unsafe-eval`. Will be removed in 8.8.0.] Set this to `false` to add the https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#unsafe_eval_expressions[`unsafe-eval`] source expression to the `script-src` directive. *Default: `true`*
```

What do y'all think about that?

Reply:
@watson I think this is the way.